gateway-developer-plugin/src/main/java/com/ca/apim/gateway/cagatewayconfig/BuildEnvironmentBundleTask.java

```diff
@@ -93,13 +93,15 @@ public void perform() {
         if(configuredFolder != null){
             bundleEnvironmentValues = environmentConfigurationUtils.loadConfigFolder(configuredFolder);
         }
-        final String envBundleFileName = getEnvBundleFilename(getProject().getName(), getProject().getVersion().toString());
+        final String configurationName = configName != null ? removeAllSpecialChars(configName.get()) : EMPTY;
```

Comment:
You can avoid `removeAllSpecialChars()` as this is applied in `getEnvBundleFilename()`

Reply:
its  already removed from getEnvBundleFileName()

Reply:
Its there at line 130.

Reply:
yeah, missed it